include/swift/AST/ProtocolConformance.h

```diff
@@ -546,6 +555,9 @@ class NormalProtocolConformance : public RootProtocolConformance,
   /// NominalTypeDecl that declared the conformance.
   DeclContext *Context;
 
+  /// The global actor isolation for this conformance, if there is one.
+  TypeExpr *globalActorIsolation = nullptr;
```

Comment:
Is this null most of the time? You can use a single bit to record the presence of this attribute, while storing the actual value in the request evaluator cache only when present.

Reply:
It's NULL most of the time. We could have a put and put it an ASTContext-managed side table, since it needs to be there at the point when the NormalProtocolConformance is created.